doc/src/base/collections.md

```diff
@@ -235,6 +236,61 @@ Partially implemented by:
   * [`Iterators.Pairs`](@ref)
 
 ## Set-Like Collections
+A [`Set`](@ref) is a collection of elements with no duplicated elements and keys/indices (unlike `Array` or `Dict`). So in a `Set` you can have only one of each element, and the order of elements isn't important.
```

Comment:
I don't really get the "keys/indices" part here. A `Set` has no keys. So perhaps something like:

```suggestion
A [`Set`](@ref) is an unordered collection of distinct elements (unlike e.g. `Array` which is ordered and can have multiple equal elements).
```

Reply:
That part of sentence was saying `Set` doesn't have `key` or `indices`